collector.go

```diff
@@ -50,10 +56,64 @@ func NewCollector(config CollectorConfig) (*Collector, error) {
 }
 
 func (c *Collector) Bundles() []Bundle {
-	return c.bundles
+	c.mutex.Lock()
+	defer c.mutex.Unlock()
+
+	return CopyBundles(c.bundles)
+}
+
+type CollectorEndpointResponse struct {
+	VersionBundles []Bundle `json:"version_bundles"`
 }
 
-// TODO
-func (c *Collector) Collect() error {
+func (c *Collector) Collect(ctx context.Context) error {
+	var responses [][]byte
+	{
+		var g errgroup.Group
+		responses = make([][]byte, len(c.endpoints))
+
+		for i, e := range c.endpoints {
+			i, e := i, e
+
+			g.Go(func() error {
+				res, err := c.restClient.NewRequest().Get(e.String())
```

Comment:
Do we have to pull `go-resty` package with all dependencies to make that call? I'd really prefer to use `http` package here, unless there is really strong benefit in using some wrapper.

Main reason is dependencies. I guess versionbundle will be imported by every operator we have, so I'd like to keep it lean.

Reply:
I am not fuzzed about this. See the list of imports being used by the lib: https://godoc.org/github.com/go-resty/resty?imports. Nothing new will be added to the operators.

Reply:
Cool I didn't see that tool.